trestle/common/list_utils.py

```diff
@@ -79,7 +79,33 @@ def delete_item_from_list(item_list: List[Any], value: Any, key: Callable[[Any],
     return item_list
 
 
+def get_item_from_list(
+    item_list: Optional[List[Any]], value: Any, key: Callable[[Any], Any], remove: bool = False
+) -> Any:
+    """Get item from list if present."""
```

Comment:
Please provide description for the function args as well

Reply:
Done